src/presets/vercel.ts

```diff
@@ -38,6 +40,8 @@ export const vercel = defineNitroPreset({
         handler: "index.mjs",
         launcherType: "Nodejs",
         shouldAddHelpers: false,
+        ...(customMemory && { memory: customMemory }),
+        ...(customMaxDuration && {maxDuration: customMaxDuration}),
```

Comment:
TIP: `JSON.stringify({ test: undefined })` is `{}`. We can simply allow undefined to be set

Reply:
Ah, thanks for the tip! Will change them to write that way.